scipy/linalg/tests/test_lapack.py

```diff
@@ -1645,3 +1645,115 @@ def test_getc2_gesc2():
         else:
             assert_array_almost_equal(desired_cplx.astype(dtype),
                                       x/scale, decimal=4)
+
+
+def test_gttrf_gttrs():
+    np.random.seed(42)
+    for index, dtype in enumerate(DTYPES):
```

Comment:
e.g. `# For each data type, use `?gttrf` and `?gttrs` to solve a random linear system`

Reply:
Will do!